### PR TITLE
Set commiter name and address for rebases

### PR DESCRIPTION
**Pull request by martin-v**

### CODE REVIEW

File: src/main/java/org/retest/rebazer/service/RebaseService.java [hunk 1]
```diff
@@ -49,6 +49,9 @@ private void setupRepo( final RepositoryConfig repoConfig ) {
 		final File repoFolder = repoFolder( repoConfig );
 		final Git localRepo = setupLocalGitRepo( credential, repoFolder, repoConfig.getUrl() );
 
+		localRepo.getRepository().getConfig().setString( "user", null, "name", "rebazer" );
+		localRepo.getRepository().getConfig().setString( "user", null, "email", "rebazer@retest.de" );
```
Comment (diba1013) [inline src/main/java/org/retest/rebazer/service/RebaseService.java]: Is this tested?

Comment (diba1013) [inline src/main/java/org/retest/rebazer/service/RebaseService.java]: Isn't this set globally?

Comment (martin-v) [reply src/main/java/org/retest/rebazer/service/RebaseService.java]: Yes manually

Comment (martin-v) [reply src/main/java/org/retest/rebazer/service/RebaseService.java]: No, with jgit not possible afaik

**Pull request merged**